css/style.css

```diff
@@ -350,8 +350,8 @@ video {
   margin-left: 14.5px;
```

Comment:
Spacing between dots are not right, change 14.5 to 11px. Also remove padding-left: 0%